%YAML:1.0
mm_per_unit: 1.0
corners:
 - [[0, 55, 0], [55, 55, 0], [55, 0, 0], [0, 0, 0]]
 - [[65, 55, 0], [120, 55, 0], [120, 0, 0], [65, 0, 0]]
 - [[130, 55, 0], [185, 55, 0], [185, 0, 0], [130, 0, 0]]
 - [[0, 120, 0], [55, 120, 0], [55, 65, 0], [0, 65, 0]]
 - [[65, 120, 0], [120, 120, 0], [120, 65, 0], [65, 65, 0]]
 - [[130, 120, 0], [185, 120, 0], [185, 65, 0], [130, 65, 0]]
 - [[0, 185, 0], [55, 185, 0], [55, 130, 0], [0, 130, 0]]
 - [[65, 185, 0], [120, 185, 0], [120, 130, 0], [65, 130, 0]]
 - [[130, 185, 0], [185, 185, 0], [185, 130, 0], [130, 130, 0]]
 - [[0, 250, 0], [55, 250, 0], [55, 195, 0], [0, 195, 0]]
 - [[65, 250, 0], [120, 250, 0], [120, 195, 0], [65, 195, 0]]
 - [[130, 250, 0], [185, 250, 0], [185, 195, 0], [130, 195, 0]]
ids:
 - 109
 - 100
 - 108
 - 103
 - 102
 - 107
 - 105
 - 101
 - 110
 - 104
 - 111
 - 106
